packages/wdio-browserstack-service/src/service.ts

```diff
@@ -276,9 +280,9 @@ export default class BrowserstackService implements Services.ServiceInstance {
         if (!passed) {
             this._failReasons.push((error && error.message) || 'Unknown Error')
         }
+        await this._accessibilityHandler?.afterTest(this._suiteTitle, test)
```

Comment:
why changing order?

Reply:
So that the data being populated for saving Accessibility test results is correctly populated ([ref](https://github.com/amaanbs/webdriverio/pull/15/files#diff-c8d3e4c2db465ea0014cefa7aa1258e4ecac3b37a4a53a94cb27f6d4dab65993R262)) (specifically test uuid)